homeassistant/helpers/entity_registry.py

```diff
@@ -103,6 +103,7 @@ class RegistryEntry:
     )
     entity_category: str | None = attr.ib(default=None)
     icon: str | None = attr.ib(default=None)
+    id: str = attr.ib(factory=uuid_util.random_uuid_hex)
```

Comment:
For areas we set the ID to be based on the name (and then it can never change again).

Is that something we want to consider?

Reply:
entities already have an entity_id & a name. Giving it a unique ID, prevents the "i want to change it, as it isn't fitting anymore" problem/question maybe? 🤷